docs/content/en/release-notes/_index.md

```diff
@@ -38,15 +38,15 @@ Release date: 27-August-2021
 The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
 
 * [Apache Cassandra](https://cassandra.apache.org/) - the deployed version depends on the configured `cassandra.version` setting:
-  * 4.0.0 (default)
+  * 4.0.1 (default)
   * 3.11.11
   * 3.11.10
   * 3.11.9
   * 3.11.8
   * 3.11.7
-* DataStax Kubernetes Operator for Apache Cassandra ([cass-operator](https://github.com/k8ssandra/cass-operator)) 1.7.1
+* [cass-operator](https://github.com/k8ssandra/cass-operator)) 1.8.0.
 * Management API for Apache Cassandra ([MAAC](https://github.com/datastax/management-api-for-apache-cassandra)) 0.1.27
-* [Stargate](https://github.com/stargate/stargate) 1.0.29
+* [Stargate](https://github.com/stargate/stargate) 1.0.40
 * Metric Collector for Apache Cassandra ([MCAC](https://github.com/datastax/metric-collector-for-apache-cassandra)) 0.2.0
 * kube-prometheus-stack 12.11.3 [chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
```

Comment:
We're bumping this up to v20.0.1 -- it's probably worth noting that we are remaining with version 7.5.11 of Grafana specifically to preserve the inclusion of the last ASL version of Grafan: 
 https://github.com/k8ssandra/k8ssandra/pull/1175/files#diff-bb025298fadbc3d3f32ec53fad275576fd87b5ed47e1aa66f6363c56a488df7eR842

Reply:
Updated - thanks.